docs/src/main/sphinx/release/release-433.md

```diff
@@ -0,0 +1,12 @@
+# Release 433 (8 Nov 2023)
+
+## General
+
+* Fix query failure when reading ORC files with a `DECIMAL` column that
+  contains only null values. ({issue}`19636`)
+
+## Hive connector
+
+* Add support for changing a column's type from `boolean` to `varchar`. ({issue}`19571`)
```

Comment:
in schema evolution or where?

Reply:
The current sentence is misleading as the connector doesn't support changing column types. The PR allows reading tables that Hive or other query engines changed the column type from boolean to varchar.